vulture/core.py

```diff
@@ -425,6 +448,15 @@ def is_identifier(s):
                 if is_identifier(attr):
                     self.used_attrs.add(attr)
 
+
+    def visit_While(self, node):
+        if condition_is_unsatisfiable(node.test):
+            self._define(self.unreachable_code, 'while', node.lineno,
+                size=lines.count_lines(node),
+                message="unsatisfiable 'while' condition",
+                confidence=DEFAULT_CONFIDENCE)
```

Comment:
This also uncovers an interesting thing, how are we gonna whitelist false positives. I think, it would need a no check comment (like `#noqa`).